File: VB/WinSolution.Module/TestTreeObject.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Xpo
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.BaseImpl

Namespace WinSolution.Module
	<DefaultClassOptions> _
	Public Class TestTreeObject
		Inherits HCategory
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub
	End Class
End Namespace

File: CS/WinSolution.Module/TestTreeObject.cs
using DevExpress.Xpo;
using DevExpress.Persistent.Base;
using DevExpress.Persistent.BaseImpl;

namespace WinSolution.Module {
    [DefaultClassOptions]
    public class TestTreeObject : HCategory {
        public TestTreeObject(Session session) : base(session) { }
    }
}
